packages/ui/src/2_molecules/Tabs/Tabs.tsx

```diff
@@ -6,11 +6,11 @@ import styles from './Tabs.module.css';
 import { TabSize, TabType } from './Tabs.types';
 import { Tab } from './components/Tab/Tab';
 
-interface ITabItem {
+export interface ITabItem {
```

Comment:
Why did you export it? For future use in our dapp?

Reply:
Yes, might be needed in the dapp